mappings/androidx-mapping.csv

```diff
@@ -2737,12 +2737,12 @@ Android.Support.Design.Widget,BottomSheetDialog,Google.Android.Material.BottomSh
 Android.Support.Design.Widget,BottomNavigationView,Google.Android.Material.BottomNavigation,BottomNavigationView,Xamarin.Android.Support.Design,Xamarin.Google.Android.Material,android.support.design.widget,BottomNavigationView,com.google.android.material.bottomnavigation,BottomNavigationView,
 Android.Support.Design.Widget,BottomNavigationView.IOnNavigationItemReselectedListener,Google.Android.Material.BottomNavigation,BottomNavigationView.IOnNavigationItemReselectedListener,Xamarin.Android.Support.Design,Xamarin.Google.Android.Material,android.support.design.widget,BottomNavigationView.OnNavigationItemReselectedListener,com.google.android.material.bottomnavigation,BottomNavigationView.OnNavigationItemReselectedListener,
 Android.Support.Design.Widget,BottomNavigationView.IOnNavigationItemReselectedListenerInvoker,Google.Android.Material.BottomNavigation,BottomNavigationView.IOnNavigationItemReselectedListenerInvoker,Xamarin.Android.Support.Design,Xamarin.Google.Android.Material,android.support.design.widget,BottomNavigationView.OnNavigationItemReselectedListener,com.google.android.material.bottomnavigation,BottomNavigationView.OnNavigationItemReselectedListener,
-Android.Support.Design.Widget,BottomNavigationView.NavigationItemReselectedEventArgs,Google.Android.Material.BottomNavigation,BottomNavigationView.NavigationItemReselectedEventArgs,Xamarin.Android.Support.Design,Xamarin.Google.Android.Material,,,,,
-Android.Support.Design.Widget,BottomNavigationView.IOnNavigationItemReselectedListenerImplementor,Google.Android.Material.BottomNavigation,BottomNavigationView.IOnNavigationItemReselectedListenerImplementor,Xamarin.Android.Support.Design,Xamarin.Google.Android.Material,mono.android.support.design.widget,BottomNavigationView_OnNavigationItemReselectedListenerImplementor,mono.com.google.android.material.bottomnavigation,BottomNavigationView_OnNavigationItemReselectedListenerImplementor,WARNING: Unable to find a Java mapping, so took a guess.
+,,,,,,,,,,WARNING: Unable to find AndroidX type for .NET type Android.Support.Design.Widget.BottomNavigationView.NavigationItemReselectedEventArgs.
```

Comment:
Are these WARNINGs intentional?

Reply:
There is no native type for `*EventArgs`, so I guess expected.